bibliopixel/project/types/defaults.py

```diff
@@ -3,6 +3,8 @@
 FIELD_TYPES = {
     'c_order': channel_order,
     'color': color,
+    'bgcolor': color,
```

Comment:
LOL...I'm probably going to remove that. It doesn't actually work in the remote section and I don't actually want it to.